tracer/src/Datadog.Trace/Iast/SourceType.cs

```diff
@@ -54,6 +55,7 @@ internal static string GetString(SourceTypeName value)
             SourceTypeName.CookieName => "http.request.cookie.name",
             SourceTypeName.CookieValue => "http.request.cookie.value",
             SourceTypeName.RequestUri => "http.request.uri",
+            SourceTypeName.Database => "database",
```

Comment:
Do we need this source type at this point? I have seen that it does not seem to be defined in https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json

If we are going to add it, we should add it there as well so all the libraries use the same name.

Reply:
It came from the Hdiv reference implementation, and I pushed it to be prepared, but I can ignore that line until that source exists